### PR TITLE
Merge Mobile rework branch into master

### PR DESCRIPTION
**Pull request by BillWagner**
sniff tests locally, but some hand merging was required

/cc @brunck

### CODE REVIEW

**Comment by tonysurma** [comment]
@BillWagner doesn't build on appveyor.  Error about services.csproj:

_Project file "Services.csproj" is not a valid MSBuild project file which must be an XML._

File: crisischeckin/Services/Services.csproj [hunk 1]
```diff
@@ -98,6 +98,10 @@
     </Compile>
     <Compile Include="Interfaces\IApiService.cs" />
     <Compile Include="Interfaces\IOrganizationService.cs" />
+<<<<<<< HEAD
```
Comment (tonysurma) [inline crisischeckin/Services/Services.csproj]: Looks like a few of the merge 'comments' are still in this file

File: crisischeckin/Models/CrisisCheckin.cs [hunk 2]
```diff
@@ -46,6 +46,12 @@ public class CrisisCheckin : DbContext
         public DbSet<Resource> Resources { get; set; }
 
         public DbSet<ResourceType> ResourceTypes { get; set; }
+<<<<<<< HEAD
```
Comment (tonysurma) [inline crisischeckin/Models/CrisisCheckin.cs]: and here too

File: crisischeckin/Models/Models.csproj [hunk 3]
```diff
@@ -66,6 +66,14 @@
     <Compile Include="Commitment.cs" />
     <Compile Include="Contact.cs" />
     <Compile Include="CrisisCheckin.cs" />
+<<<<<<< HEAD
```
Comment (tonysurma) [inline crisischeckin/Models/Models.csproj]: and here

File: crisischeckin/Models/Models.csproj [hunk 4]
```diff
@@ -212,6 +220,12 @@
     <EmbeddedResource Include="Migrations\201604191341494_AddVerifiedFlagToOrganizations.resx">
       <DependentUpon>201604191341494_AddVerifiedFlagToOrganizations.cs</DependentUpon>
     </EmbeddedResource>
+<<<<<<< HEAD
```
Comment (tonysurma) [inline crisischeckin/Models/Models.csproj]: and here

File: crisischeckin/Models/Organization.cs [hunk 5]
```diff
@@ -31,6 +31,10 @@ public class Organization
         public virtual IList<Resource> Resources { get; set; }
 
         public virtual IList<Person> Persons { get; set; }
+<<<<<<< HEAD
```
Comment (tonysurma) [inline crisischeckin/Models/Organization.cs]: Here

File: crisischeckin/Models/Person.cs [hunk 6]
```diff
@@ -25,7 +25,12 @@ public class Person
         public int? OrganizationId { get; set; }
         
         public virtual Organization Organization { get; set; }
+<<<<<<< HEAD
```
Comment (tonysurma) [inline crisischeckin/Models/Person.cs]: Here

File: crisischeckin/Services/DataService.cs [hunk 7]
```diff
@@ -3,6 +3,10 @@
 using Services.Interfaces;
 using System;
 using System.Linq;
+<<<<<<< HEAD
```
Comment (tonysurma) [inline crisischeckin/Services/DataService.cs]: This file as well

File: crisischeckin/Services/Interfaces/IDataService.cs [hunk 8]
```diff
@@ -1,5 +1,9 @@
 using Models;
 using System.Linq;
+<<<<<<< HEAD
```
Comment (tonysurma) [inline crisischeckin/Services/Interfaces/IDataService.cs]: and this one

File: crisischeckin/Services/Services.csproj [hunk 9]
```diff
@@ -113,6 +117,10 @@
     <Compile Include="Interfaces\IDisaster.cs" />
     <Compile Include="Interfaces\IVolunteerService.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
+<<<<<<< HEAD
```
Comment (tonysurma) [inline crisischeckin/Services/Services.csproj]: here

**Comment by tonysurma** [comment]
Couldn't get into code but marked where there were merge 'comments' left that is breaking build etc.

**Comment by BillWagner** [comment]
Really weird. GH desktop pushed the pre merge version. Will fix tonight

**Comment by brunck** [comment]
So this latest commit removed the lines that GitHub put in to show the conflicts, but it doesn't seem like the conflicts were resolved, just stacked one on top of the other. Are we sure that's what we want as a resolution?

**Comment by brunck** [comment]
Looking at the AppVeyor build results, it seems like things aren't resolved correctly. The removal of EntitiesController and other Breeze stuff isn't there; the deletions aren't applied.

**Comment by brunck** [comment]
If you'd like me to make a new PR to mobile-rework, if you merge that PR as a new commit (don't rebase), I think it will merge the master commits into mobile-rework correctly. 
I strongly suggest merging PRs via the GitHub interface; that will create a merge commit and won't rebase.

**Comment by brunck** [comment]
I suggest we delete this PR and merge PR #661.

**Comment by brunck** [comment]
I also suggest we delete the "mobile" branch. It is out of synch with master (unlike the branch in PR #661) and all the changes are in that PR that are duplicated in "mobile."

**Comment by BillWagner** [comment]
abandon this. It's just going

**Pull request closed without merging**